Update release notes for Keycloak 26.1.0 with new community additions (#35590)
keycloak/keycloak: docs/documentation/release_notes/topics/26_1_0.adoc

Alternatively, you can disable dark mode support for the built-in Keycloak themes on a per-realm basis by turning off the "Dark mode" setting under the "Theme" tab in the realm settings.

> = Authorization Code Binding to a DPoP Key
> 
> There is added support for https://datatracker.ietf.org/doc/html/rfc9449#section-10[Authorization Code Binding to a DPoP Key] including support for the DPoP with Pushed Authorization Requests.

Reviewer: ```suggestion
= Authorization Code Binding to a DPoP Key

Support now exists for https://datatracker.ietf.org/doc/html/rfc9449#section-10[Authorization Code Binding to a DPoP Key] including support for the DPoP with Pushed Authorization Requests.
```

ifeval::[{project_community}==true]
Many thanks to https://github.com/tnorimat[Takashi Norimatsu] for the contribution.
endif::[]

> = Allow to create certificates for generated EC keys

Reviewer: ```suggestion
= Option to create certificates for generated EC keys
```

> There is an option `Generate certificate` added for EC-DSA and Ed-DSA key providers. When the generated key is created by realm administrator, there might be also
> certificate generated for this key. The certificate informations are available in the admin console and in the JWK representation of this key, which is available
> from JWKS endpoint with the realm keys.

Reviewer: ```suggestion
A new option, `Generate certificate`, exists for EC-DSA and Ed-DSA key providers. When the generated key is created by realm administrator, a 
certificate might be generated for this key. The certificate information is available in the Admin Console and in the JWK representation of this key, which is available
from JWKS endpoint with the realm keys.
```

ifeval::[{project_community}==true]
Many thanks to https://github.com/Captain-P-Goldfish[Pascal Knüppel] for the contribution.
endif::[]

= Minimum ACR Value for the client

> The option `Minimum ACR value` is added as a configuration option on the realm OIDC clients. This is an enhancement related to step-up authentication, which makes it possible
> to enforce minimum ACR level when login to the particular client.

Reviewer: ```suggestion
The option `Minimum ACR value` is added as a configuration option on the realm OIDC clients. This addition is an enhancement related to step-up authentication, which makes it possible
to enforce minimum ACR level when logging in to the particular client.
```
